cupy/linalg/util.py

```diff
@@ -33,8 +33,10 @@ def _check_cusolver_dev_info_if_synchronization_allowed(routine, dev_info):
     # documentation.
     assert isinstance(dev_info, core.ndarray)
     assert dev_info.size == 1
-
-    config_linalg = cupyx._ufunc_config.config.linalg
+    try:
+        config_linalg = cupyx._ufunc_config.config.linalg
+    except AttributeError:
+        config_linalg = 'ignore'
```

Comment:
Should assign `cupyx._ufunc_config.config.linalg` as well.
```suggestion
        config_linalg = cupyx._ufunc_config.config.linalg = 'ignore'
```